app/views/bills/_form.html.erb

```diff
@@ -22,11 +34,13 @@
     <%= f.text_field :usage, :id => "usage" %>
   </div>
 
-  <%= f.fields_for :units do |units_form| %>
-    <div class="field">
-      <%= units_form.label :number_occupants, "Number of Occupants in Your Home or Apartment" %><br>
-      <%= units_form.text_field :number_occupants, :id => "occupants" %>
-    </div>
+  <% if @show_occupants %>
```

Comment:
Could you just do `unless user_signed_in? && current_user.unit.present?` here instead of having to do that in the controller and then passing it to a new variable?